otb-tracker/bin

```diff
@@ -16,39 +16,99 @@ export OTB_TRACKER_INTERFACE
 export OTB_TRACKER_CALL
 export OTB_TRACKER_HOST
 export OTB_TRACKER_TIMEOUT
-export OTB_TRACKER_RETRY
+export OTB_TRACKER_TRIES
 export OTB_TRACKER_INTERVAL
 export OTB_TRACKER_STATUS
 export OTB_TRACKER_RESULT
 export OTB_TRACKER_DEVICE
-export OTB_TRACKER_INTERFACE_IP
-export OTB_TRACKER_INTERFACE_GATEWAY
+export OTB_TRACKER_DEVICE_IP
+export OTB_TRACKER_DEVICE_GATEWAY
+
+# set constants for rto updating
+# we've changed the default values of the RFC to K=3 and beta=0.25 instead of K=4 and beta=0.125
+# this allows us to lower the gap between rtt and rto when the latency increases quickly as our measurements are less frequent than the standard timeout computation of the TCP stack
```

Comment:
That's a looooooooooooooooooooooooooonnnnnnnnnnnngggggggggggggg line !
You should wrap this up